### ISSUE TITLE
Incompatible with PHPStorm 2020.2

### ISSUE THREAD

**Issue by Spurlos** [npvinhsd/VueRouterIdeaPlugin#2]
After update to latest IDE version, this goes to the event log.

```
31/07/2020
13:52	Plugin Error
				Plugin "Vue Router AutoCompletion" is incompatible (until build 201.SNAPSHOT < PS-202.6397.115).
```

Plugin still may work, but it looks like its out of version constraints.

**Comment by npvinhsd** [comment]
Please give me the exactly of your PHPStorm version. PhpStorm .2020.3 has not released yet. And this plugin does not work in EAP version.

**Comment by Spurlos** [comment]
I made a mistake there. Its actually 2020.2 - the latest release which happened this week.

I'll edit the title of the issue right now.

**Comment by npvinhsd** [comment]
Hi, I have updated the fix that supports 2020.2. Please update to latest version
